e2e/tests/ui/features/@search/search.step.ts

```diff
@@ -0,0 +1,275 @@
+import { SearchPage } from "../../helpers/SearchPage";
```

Comment:
With [Carlos' page object PR](https://github.com/guacsec/trustify-ui/pull/658) merged, we should probably consider these classes outdated and use the ones from that merge request if possible to avoid redundancy.